config/webpack.config.dev.js

```diff
@@ -74,7 +74,8 @@ module.exports = {
     // https://github.com/facebookincubator/create-react-app/issues/253
     fallback: paths.nodePaths,
     // These are the reasonable defaults supported by the Node ecosystem.
-    extensions: ['.js', '.json', ''],
+    // We also include JSX as a common component filename extension.
```

Comment:
I would specifically add that we don’t recommend using it but some tools rely on it, and link to the issue.